pkgs/by-name/we/wemeet/package.nix

```diff
@@ -0,0 +1,130 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  makeWrapper,
+
+  nss,
+  xorg,
+  desktop-file-utils,
+  libpulseaudio,
+  libgcrypt,
+  dbus,
+  systemd,
+  udev,
+  libGL,
+  fontconfig,
+  freetype,
+  openssl,
+  wayland,
+  libdrm,
+  harfbuzz,
+  openldap,
+  curl,
+  nghttp2,
+  libunwind,
+  alsa-lib,
+  libidn2,
+  rtmpdump,
+  libpsl,
+  libkrb5,
+  xkeyboard_config,
+  zenity,
+}:
+
+stdenv.mkDerivation {
+  pname = "wemeet";
+  version = "3.19.2.400";
+
+  src = fetchurl {
+    # Source deb pack comes from the official link:
+    # https://meeting.tencent.com/download/
+    url = "https://updatecdn.meeting.qq.com/cos/fb7464ffb18b94a06868265bed984007/TencentMeeting_0300000000_3.19.2.400_x86_64_default.publish.officialwebsite.deb";
+    hash = "sha256-PSGc4urZnoBxtk1cwwz/oeXMwnI02Mv1pN2e9eEf5kE=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  buildInputs = [
+    nss
+    xorg.libX11
+    desktop-file-utils
+    libpulseaudio
+    libgcrypt
+    dbus
+    systemd
+    udev
+    libGL
+    xorg.libSM
+    xorg.libICE
+    xorg.libXtst
+    libGL
+    fontconfig
+    freetype
+    openssl
+    wayland
+    libdrm
+    harfbuzz
+    openldap
+    curl
+    nghttp2
+    libunwind
+    alsa-lib
+    libidn2
+    rtmpdump
+    libpsl
+    libkrb5
+    xkeyboard_config
+    zenity
```

Comment:
I think there is no need to add `zenity` to `buildInputs`, it is only used in `substituteInPlace`.

Reply:
fixed